mission-vision.md

```diff
@@ -0,0 +1,30 @@
+# OpenFeature Mission and Vision
+
+## Mission
+
+To improve the software development lifecycle, no matter the size of the project, by standardizing feature flagging for developers.
+
+## Vision
+
+A world where feature flagging is a core principle of the software development lifecycle, enabling developers to test and release high quality software with a high degree of safety and confidence.
+
+OpenFeature can achieve this vision by:
+
+### Vendor neutrality
+
+OpenFeature aims to be the primary mechanism an [application author][glossary-app-auth] interacts with when using feature flags in code by standardizing the developer facing interfaces in a vendor neutral way. This provides flexibility for the [application integrators][glossary-app-int] to choose the tooling that best fits their current requirements, while avoiding code-level lock-in.
+
+### Extensibility
+
+OpenFeature should be extensible while still being easy to work with. It should be quick to get started with, while providing [application authors][glossary-app-auth] the flexibility to customize the functionality based on their business requirements and workflows.
```

Comment:
> OpenFeature should be extensible...

We say "OpenFeature is" elsewhere, and "OpenFeature should" here.

I think there's a tenancy to want to use future tense here because were just starting, but that wont be the case forever. We should perhaps say something like "OpenFeature values/prioritizes extensibility", and keep things present tense.

Reply:
I should use future tense everywhere since this is a subset of the vision.

Reply:
I think consistency is important... I feel like present tense is the best choice, but maybe I'm wrong about that.

Reply:
I ended up using present tense. Please let me know if you notice any inconsistencies.